### DISCUSSION TITLE
How to remove this Taskbar X downgrade

### DISCUSSION THREAD

**Discussion by edwfelt** [valinet/ExplorerPatcher#1422] [Q&A] [answered]
https://github.com/valinet/ExplorerPatcher/releases/tag/22621.1555.55.2_6fb998e

Not a tech person, I apologize if this is a dumb question.

I installed this update on my laptop for Taskbar X. However, I cannot remove it by deleting the file or changing the file permission. Would downloading the latest Windows Update resolve this issue? If no, I hope someone can show me the steps on how to remove the update. Thanks ^^

**Comment by pyrates999** [comment]
That is a pre-release.  You should uninstall it and only install releases unless you want to help in bug testing.  Here's the latest release: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.1555.55.1_a95a688

**Reply by edwfelt** [reply]
Oooh I see! Could you show me where can I uninstall the previous version? The only option I saw was to delete the files. I have downloaded the latest version but my taskbar is still glitching, some parts like the search bar is invisible despite refreshing or restarting PC. 

<img width="960" alt="Screenshot (497)" src="https://user-images.githubusercontent.com/129827160/236625250-5b0de046-6d9e-4d49-a777-e8e8bf858084.png">

**Comment by pyrates999** [comment, marked as answer]
Here's how to uninstall it: https://github.com/valinet/ExplorerPatcher/wiki#uninstalling

**Reply by edwfelt** [reply]
Thank you so much for the link! I'm not sure why my taskbar is still glitching but that's for another day, have a good day kind sir ^^
